PP_APP/README.md

```diff
@@ -0,0 +1,59 @@
+# HOW TO SETUP AND RUN THE API FOR THE FIRST TIME
+
+
+1. Open up the terminal in this directory and `cd server`
+2. `python manage.py makemigrations`
+3. `python manage.py migrate`
+4. `python manage.py createsuperuser`
+    - So you can have control over the admin page.
+    - This will prompt you for email, username (just put your first name), password confirm password.
+    - The password can be something small like "193". However, if you do have small somthing small like that, then you will have to 'override' when Django asks if you want to.
+5. `python manage.py runserver`
+
+If you have already made the migrations, skip step 2 and 3.
+If you have already made a superuser, skip step 4.
+
+The API should be running!
+
+Goto admin page: http://localhost:8000/admin/ and login to have control.
+
+Use POSTMAN to make UNAUTHENTICATED requests:
+
+1. http://localhost:8000/api/create_hello - append to body `name` and `msg` field and make a `POST` request.
+2. http://localhost:8000/api/get_hello - make a `GET` request, this will query everything in the database table `hello_rest`.
+
+Use POSTMAN to make AUTHENTICATED requests:
+
+0. Register a user by making a `POST` request. 
+    - enter http://localhost:8000/accounts/register into the URL field. 
+    - Click on 'Body' below the URL field. The `key`s are `email`, `username`, `password`, `password2`. Enter `value`s of your choice. And press 'Send'.
+    - You will recive a JSON that looks like this:
+        ```
+        {
+        "response": "Account Registered.",
+        "email": "yoyo@abc.com",
+        "username": "Yeeet",
+        "token": "7a39fe5f41f840a2302c8adaf5f8ac8309886aa5"
+        }
+        ```
+    - Save that token.
+1. `POST` with http://localhost:8000/api/upload - with 'Body' keys and values.
+    - The `key`s are: `packager`, **`top_img`, **`side_img`
+    - ** make sure you select 'File' in the Key drop down meny for these.
+    - Click on Headers
+        - Key: type in `Authorization`
+        - Value: type in `token 7a39fe5f41f840a2302c8adaf5f8ac8309886aa5`
```

Comment:
The value used by the token is whatever token is returned after the new account is created. It is not necessarily this token.

Reply:
Yup.